README.md

```diff
@@ -78,37 +78,19 @@ Read more at [Hyperswitch docs](https://docs.hyperswitch.io/).
 ### 1. Local Setup
 
 You can run Hyperswitch on your system using Docker compose after cloning this repository. 
+We recommend using Docker Desktop for Windows and Mac OS (Or Orbstack). On Linux, you can install Docker Engine directly.
 
 ```shell
 git clone --depth 1 --branch latest https://github.com/juspay/hyperswitch
 cd hyperswitch
 docker compose up -d
+./scripts/docker_output.sh
```

Comment:
Nit:

```suggestion
scripts/docker_output.sh
```

Can you run `chmod +x scripts/docker_output.sh` and then commit the file?